web/vital_records/forms.py

```diff
@@ -200,21 +210,20 @@ class OrderInfoForm(forms.ModelForm):
     )
     zip_code = forms.CharField(
         label="Zip code",
-        max_length=5,
+        max_length=10,
         required=True,
         widget=forms.TextInput(
             attrs={
                 "class": "form-control",
                 "inputmode": "numeric",
-                "minlength": "5",
-                "maxlength": "5",
+                "pattern": "[\d]{5}(-[\d]{4})?",
```

Comment:
I think this should use raw Python regex string? I'm seeing a warning in the console:

```log
/cdt/app/web/vital_records/forms.py:219: SyntaxWarning: invalid escape sequence '\d'
  "pattern": "[\d]{5}(-[\d]{4})?"
```

Marking it as a raw regex string (just prepend the `r` character before the string):

```python
"pattern": r"[\d]{5}(-[\d]{4})?",
```

See https://docs.python.org/3/library/re.html#raw-string-notation